README.md

```diff
@@ -109,6 +109,25 @@ Example of using it in controller
     {
 ```
 
+## Comment using attributes (>= php 8)
+Extra comments can also be added to controller methods using attributes. Currently supported attributes are
+- `Rakutentech\LaravelRequestDocs\Attributes\RouteDescription`
+- `Rakutentech\LaravelRequestDocs\Attributes\RouteDeprecated`
+
+```php
+#[RouteDescription('Title', 'Description')]
```

Comment:
Not sure if we would want to include it to be honest. It seems a lot more work to figure out what and how to use this feature, when a simple ``@lrd:start`` works fine. I am sorry I'd have to decline this pull request.

Reply:
>when PHP attributes have sunk into the community a bit more.

You have put it right and thanks or understanding. Yes, most likely this will be a revisit. Let's see if someone lands on this issue in future and we can rework on it.